sirepo/package_data/static/js/srw.js

```diff
@@ -1595,18 +1595,14 @@ SIREPO.app.directive('appHeader', function(appState, panelState, srwService) {
               ${rightNav}
             </div>
             <div data-ng-if="srwService.isApplicationMode(\'default\')">
-              <div data-app-header-brand="" data-app-url="{{ ::appURL() }}"></div>
```

Comment:
This needs to stay

Reply:
df44df69b0cf6074618593a4c872ea9c2213cc9a